lutris/util/resources.py

```diff
@@ -73,7 +75,13 @@ def fetch_icons(game_slugs, callback=None):
         callback(updated_slugs)
 
 def udpate_desktop_icons():
-    os.system("xdg-icon-resource forceupdate")
+    # Update Icon for GTK+ desktop manager
+    gtk_update_icon_cache = shutil.which("gtk-update-icon-cache")
+    if (gtk_update_icon_cache):
+        os.system("gtk-update-icon-cache -tf %s"
+        % os.path.join(GLib.get_user_data_dir(), 'icons', 'hicolor'))
```

Comment:
I'll fix that later but that is a weird way of cutting lines! X)

Reply:
Oh sorry. I'm not from a python background.

Reply:
It's ok, the code is good. I suggest you integrate Flake8 in whatever editor you're using and if you have no Flake8 warning then it's formatted correctly, and will tell you what's wrong if not